### PR TITLE
Adding worker template and docs

### PR DESCRIPTION
**Pull request by Scotsoo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,39 @@
...
+```bash
+wrangler generate netacea-cloudflare-worker https://github.com/netacea/cloudflare-worker-template-typescript
+```
+
+Ensure the `@netacea/cloudflare` package is up to date by running:
+```bash
+npm i @netacea/cloudflare@latest
+```
+
+Insert your Netacea API and Secret key into `./src/NetaceaConfig.json`. Ensure you are [logged into wrangler](https://github.com/cloudflare/wrangler#-login). Insert your wrangler credentials into the `wrangler.toml` file in the root directory (you only need to change `account_id`, `route` and `zone_id` - the rest is okay).
```
Comment (curtisjohnson1) [inline README.md:22]: I would convert this body of text into 3 bullet points so users are more likely to read/scan over the steps required

File: README.md [hunk 2]
```diff
@@ -1,2 +1,39 @@
...
+
+## ✔ Testing
+- `npm run test` - This will run a set of simple method tests against the worker (see `./tests/handler.test.ts`)
+- `npm run dev-cf` - alias for `wrangler dev` - [documentation](https://github.com/cloudflare/wrangler#-dev).
+- `npm run watch-cf` - alias for `wrangler preview --watch` - live preview of your code in a cloudflare sandbox - accessible from only your machine.
+
+## ☁ Publishing
+`npm run publish-cf` is your friend. This will publish to your cloudflare distribution
+
+## 🤢 Issues
```
Comment (rswalkden) [inline README.md:38]: Maybe use an exclamation mark like github does?
image

**Review by rswalkden — DISMISSED** [review]
LGTM. One option change suggested

File: README.md (hunk 1 above)
Comment (Scotsoo) [reply README.md]: Done! Nice one

File: README.md (hunk 2 above)
Comment (Scotsoo) [reply README.md]: Good shout, done!

**Review by rswalkden — DISMISSED** [review]

**Review by rswalkden — APPROVED** [review]

**Review by curtisjohnson1 — APPROVED** [review]
🚀 🚀

**Pull request merged**